### DISCUSSION TITLE
Grouping the components in the website

### DISCUSSION THREAD

**Discussion by Mil4n0r** [dxc-technology/halstack-react#1771] [Ideas] [answered]
In designing our categorized collection of components, we have to recognize the challenge posed by the sheer number of elements available. The goal is to create a user-friendly environment where locating specific components is intuitive and efficient. 

Categorizing components based on their functionality ensures that users, whether designers or developers, can quickly locate the elements they need for specific tasks. This promotes a smoother workflow and reduces the cognitive load associated with searching through an extensive list.

Current non-grouped list:

![image](https://github.com/dxc-technology/halstack-react/assets/33158831/9d1aa435-e44b-458f-8c79-a79c271114e4)

I have prepared a first _mockup_ of what I would consider a good way to organize the current elements:

### 1. Layout

- Accordion
- Application Layout
- Bleed
- Box
- Flex
- Header
- Grid
- Inset

### 2. Navigation
- Nav Tabs
- Paginator
- Quick Nav
- Sidenav
- Tabs

### 3. Inputs

- Checkbox
- Date Input
- Dropdown
- File Input
- Number Input
- Password Input
- Radio Group
- Select
- Slider
- Text Input
- Textarea
- Toggle Group

### 4. Action

- Button
- Link
- Wizard

### 5. Feedback 

- Alert
- Dialog
- Progress Bar
- Spinner
- Switch
- Tag

### 6. Display

- Bulleted List
- Card
- Chip
- Image
- Resultset Table
- Table
- Tag

### 7. Text
- Heading
- Paragraph
- Typography

**Comment by GomezIvann** [comment, marked as answer]
We will continue this discussion on #1950 🚀
